apps/bsp/bulk-scan-payment-processor/bulk-scan-payment-processor.yaml

```diff
@@ -6,9 +6,11 @@ spec:
   releaseName: bulk-scan-payment-processor
   values:
     java:
-      replicas: 2
+      autoscaling:
+        enabled: false
+      replicas: 0
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/bulk-scan/payment-processor:prod-663cba4-20250225105556 #{"$imagepolicy": "flux-system:bulk-scan-payment-processor"}
+      image: http://hmctspublic.azurecr.io/bulk-scan/payment-processor:pr-1219 #{"$imagepolicy": "flux-system:bulk-scan-payment-processor"}
```

Comment:
Did you mean to do this in the `perftest.yaml` patch for payment processor? This will affect all environments

Reply:
I ABSOLUTELY DID NOT THANK YOU